### PR TITLE
chore: resolve linter issues

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,25 +12,35 @@ require (
 	github.com/gin-gonic/gin v1.4.0
 	github.com/go-logr/logr v0.1.0
 	github.com/gofrs/uuid v3.3.0+incompatible
+	github.com/golangci/golangci-lint v1.31.0 // indirect
```
Comment (MatousJobanek) [inline go.mod:15]: could you please run `go mod tidy`?

**Review by MatousJobanek — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (xcoulon) [reply go.mod]: oh sure 😳 
done in 0a84976

File: pkg/configuration/configuration.go [hunk 2]
```diff
@@ -201,11 +201,6 @@ func New(configFilePath string, cl client.Client) (*Config, error) {
 	return c, nil
 }
 
-func getRegistrationEnvVarKey(key string) string {
```
Comment (tinakurian) [inline pkg/configuration/configuration.go:204]: good catch

**Review by tinakurian — APPROVED** [review]
Looks good!

File: pkg/configuration/configuration.go (hunk 2 above)
Comment (xcoulon) [reply pkg/configuration/configuration.go:204]: thanks, but credit goes to `golangci-lint` 😂

**Pull request merged**